sphinx_doc/howtos/configuration/configuring-the-controller.rst

```diff
@@ -0,0 +1,318 @@
+  |                | the robot than this distance are discarded.                                |
+  |                | This is re-evaluated each time the controller is run, so the points are not|
+  |                | permanently discared.                                                      |
+  +----------------+----------------------------------------------------------------------------+
+
+:transform_tolerance:
+  +----------------+----------------------------------------------------------------------------+
+  |**type**        | double                                                                     |
+  +----------------+----------------------------------------------------------------------------+
+  |**default**     |  0.1                                                                       |
+  +----------------+----------------------------------------------------------------------------+
+  |**units**       |  s                                                                         |
+  +----------------+----------------------------------------------------------------------------+
+  |**description** | When determining the current robot pose, if the transform data is older    |
+  |                | than this, an error is returned                                            |
+  +----------------+----------------------------------------------------------------------------+
+
+Debug Parameters
+================
+
+:debug_trajectory_details:
+  +----------------+----------------------------------------------------------------------------+
+  |**type**        | bool                                                                       |
+  +----------------+----------------------------------------------------------------------------+
+  |**default**     | false                                                                      |
+  +----------------+----------------------------------------------------------------------------+
+  |**description** | If true and if all trajectories are rejected by the critics, DWB will print|
+  |                | statistics on the console indicating which critics rejected the            |
+  |                | trajectories and for what reason.                                          |
+  +----------------+----------------------------------------------------------------------------+
+
+:publish_cost_grid_pc:
+  +----------------+----------------------------------------------------------------------------+
+  |**type**        | bool                                                                       |
+  +----------------+----------------------------------------------------------------------------+
+  |**default**     | false                                                                      |
+  +----------------+----------------------------------------------------------------------------+
+  |**description** | If true, publishes the ????? on the **cost_cloud** topic. This can be      |
```

Comment:
????? -> point cloud cost grid or voxel cost grid

Reply:
I don't think that really explains anything. I also don't believe it is just republishing the costmap data. I believe it is more correctly publishing the scores that the critics are assigning cells, but I'm not 100% certain. For instance, not all critics have a score for every cell.